src/auth/forward.h

```diff
@@ -25,6 +25,8 @@ typedef std::vector<Auth::SchemeConfig *> ConfigVector;
 
 class UserRequest;
 
+using Ttl = int32_t;
```

Comment:
The underlying type -- int32_t -- is wrong on several (probably obvious?) levels. However, I assume that this PR scope is limited to _naming_ existing int32_t uses and excludes fixing the underlying type. Thus, I am not requesting those fixes in this PR.